gobblin-yarn/src/main/java/org/apache/gobblin/yarn/YarnHelixUtils.java

```diff
@@ -46,8 +47,9 @@
  *
  * @author Yinan Li
  */
+@Slf4j
 public class YarnHelixUtils {
-
+   public static final String ADDITIONAL_YARN_CLASSPATH = "additional.yarn.classpath";
```

Comment:
Should this config be part of GobblinYarnConfigurationKeys? e.g. gobblin.yarn.additional.classpaths?